src/collection.js

```diff
@@ -483,6 +483,16 @@ class Collection {
 
     return this
   }
+
+  /**
+   *Returns the average of all collection items

```

Comment:
Please add a space between the asterisk and the word: `* Returns` 
```suggestion
   * Returns the average of all collection items
```